docs/functions/theme.rst

```diff
@@ -90,6 +90,12 @@ specific text and formatting may change.
 
    .. versionadded:: 2.0
 
+.. function:: _lp_aws_profile() -> var:lp_aws_profile
+
+   Returns :func:`_lp_analog_time` with color from :attr:`LP_COLOR_AWS_PROFILE`.
```

Comment:
```suggestion
   Returns :func:`_lp_aws_profile` with color from :attr:`LP_COLOR_AWS_PROFILE`.
```

Reply:
Apologies for the careless cut-and-paste error - I should have spotted this!